runtime/dali/Cargo.toml

```diff
@@ -174,12 +175,14 @@ std = [
 	"crowdloan-rewards/std",
 	"bonded-finance/std",
 	"dutch-auction/std",
+	"liquidations/std",
 	"vesting/std",
 	"mosaic/std",
 	"preimage/std",
 	"membership/std",
 	"system-rpc-runtime-api/std",
 	"transaction-payment-rpc-runtime-api/std",
+	"liquidations/std",
```

Comment:
seems redundant

Reply:
why? all other pallets set their std here?

Reply:
yes but line number 178 and 185 are identical , I don't understand why we need both of them?

Reply:
ahaa:)  thanks
![image](https://user-images.githubusercontent.com/757125/153165916-0ac8fcb8-02df-433f-bc44-0c06f3f95241.png)

Reply:
just was not visible in uI